src/test/resources/com/puppycrawl/tools/checkstyle/main/InputMainJavadocComment.javadoc

```diff
@@ -1,4 +1,4 @@
 <p>
 My {@code Class}
 </p>
-@see never
\ No newline at end of file
+@see never
```

Comment:
adding "\n" is not required, please revert it as it cause useless changes in expected AST.
If you want some tests to have "\n", please declare a need in PR description to let clearly understand your intend.